From: Xavier Claessens <xavier.claessens@collabora.com>
Date: Wed, 19 Sep 2018 22:36:38 -0400
Subject: Rename cxx-test.C to cxx-test.cpp

Meson checks file extension case-insentively, and compiles ".C" as C
source instead of C++.

Origin: upstream, 2.59.0, commit:ef1bcc5a0423bff1ab9bb14bdf740d27a84c5c09
---
 tests/Makefile.am  |  2 +-
 tests/cxx-test.C   | 10 ----------
 tests/cxx-test.cpp | 10 ++++++++++
 tests/meson.build  |  2 +-
 4 files changed, 12 insertions(+), 12 deletions(-)
 delete mode 100644 tests/cxx-test.C
 create mode 100644 tests/cxx-test.cpp

diff --git a/tests/Makefile.am b/tests/Makefile.am
index fba1865..9682352 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -123,7 +123,7 @@ dist-hook: $(BUILT_EXTRA_DIST)
 
 if HAVE_CXX
 installed_test_programs += cxx-test
-cxx_test_SOURCES = cxx-test.C
+cxx_test_SOURCES = cxx-test.cpp
 endif
 
 if ENABLE_TIMELOOP
diff --git a/tests/cxx-test.C b/tests/cxx-test.C
deleted file mode 100644
index c076e88..0000000
--- a/tests/cxx-test.C
+++ /dev/null
@@ -1,10 +0,0 @@
-#include <glib.h>
-#include <gmodule.h>
-#include <glib-object.h>
-#include <gio/gio.h>
-
-int
-main ()
-{
-  return 0;
-}
diff --git a/tests/cxx-test.cpp b/tests/cxx-test.cpp
new file mode 100644
index 0000000..c076e88
--- /dev/null
+++ b/tests/cxx-test.cpp
@@ -0,0 +1,10 @@
+#include <glib.h>
+#include <gmodule.h>
+#include <glib-object.h>
+#include <gio/gio.h>
+
+int
+main ()
+{
+  return 0;
+}
diff --git a/tests/meson.build b/tests/meson.build
index 162051a..7530571 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -51,7 +51,7 @@ tests = [{
   },
   'timeloop' : {},
   'cxx-test' : {
-    'source' : 'cxx-test.C',
+    'source' : 'cxx-test.cpp',
     'include_directories' : gmoduleinc,
     'dependencies' : [libgio_dep],
   },
